apps/mark-scan/backend/src/custom-paper-handler/state_machine.test.ts

```diff
@@ -249,6 +249,25 @@ describe('not_accepting_paper', () => {
   });
 });
 
+describe('eject_to_front', () => {
+  test.each([
+    {
+      description: 'paper is absent',
+      setStatus: () => setMockDeviceStatus(getDefaultPaperHandlerStatus()),
+    },
+    {
+      description: 'paper is in front input',
+      setStatus: () => setMockDeviceStatus(getPaperInFrontStatus()),
+    },
```

Comment:
Do you want coverage for the partially detected state?

Reply:
[20e077b](https://github.com/votingworks/vxsuite/pull/4857/commits/20e077b8debd58ecbe9e70ba98f9b89b18437746)